packages/ruleset-bundler/README.md

```diff
@@ -0,0 +1,15 @@
+# @stoplight/spectral-ruleset-bundler
+
+**WARNING** - for the time being, the following package is meant to be used internally.
```

Comment:
Any reason to actually publish this then? Versus just not treat it as external dep and bundle it into the other spectral packages that use it in this workspace?

Reply:
OH, I see I think - do we need to be able to use this package from our ecosystem standalone?

Reply:
> OH, I see I think - do we need to be able to use this package from our ecosystem standalone?

yeah